roles/mailcatcher/molecule/default/verify.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Verify
+  hosts: all
+  gather_facts: false
+  tasks:
+  - name: install global mail catcher
```

Comment:
@kayiwa what does this do?

Reply:
this was a test to figure out why it wasn't installing. Forgot to remove it before push. Fix with my last commit.